### PR TITLE
[hi]Localize content/en/docs/reference/glossary/cgroup.md

### PR DESCRIPTION
**Pull request by taniaduggal**
Signed-off-by: Tania Duggal <103496926+taniaduggal@users.noreply.github.com>
fixes #35600

### CODE REVIEW

**Comment by taniaduggal** [comment]
@anubha-v-ardhan Please review :)

File: content/hi/docs/reference/glossary/cgroup.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+title: सीग्रुप (cgroup)
+id: cgroup
+date: 2019-06-25
+full_link:
+short_description: >
+  Linux प्रक्रियाओं का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cgroup.md:7]:
```suggestion
  Linux प्रक्रियाओं (processes) का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।
```
I think having processes written in brakets makes this easier to understand for some people?

File: content/hi/docs/reference/glossary/cgroup.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+date: 2019-06-25
+full_link:
+short_description: >
+  Linux प्रक्रियाओं का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।
+
+aka:
+tags:
+- fundamental
+---
+Linux प्रक्रियाओं का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/reference/glossary/cgroup.md:13]:
```suggestion
Linux प्रक्रियाओं (processes) का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।
```
same as above

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Thank you @taniaduggal. Can you please squash the commits so we can get this PR ready for merge.

**Comment by divya-mohan0209** [comment]
@taniaduggal : Please sync your fork & squash commits so that we can proceed with merge.

**Comment by taniaduggal** [comment]
@divya-mohan0209 i think i already squash the commits

File: content/hi/docs/reference/glossary/cgroup.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- fundamental
+---
+Linux प्रक्रियाओं (processes) का एक समूह जो चुनिंदा रूप से संसाधनों को अलग, प्रबंधित और प्रतिबंधित करता है।
+
+<!--more--> 
+
+cgroup एक Linux कर्नेल सुविधा है जो प्रक्रियाओं के एक सेट के लिए संसाधन उपयोग (CPU, मेमोरी, डिस्क I/O, नेटवर्क) को अलग, प्रबंधित और सीमित करता है।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/glossary/cgroup.md:17]:
```suggestion
cgroup एक Linux कर्नल सुविधा है जो प्रक्रियाओं के एक समूह के लिए संसाधन उपयोग (CPU, मेमोरी, डिस्क I/O, नेटवर्क) को अलग, प्रबंधित, और सीमित करता है।
```

**Comment by divya-mohan0209** [comment]
Duplicate of #38954 
/close

**Pull request closed without merging**